### PR TITLE
New extension: Bike

### PR DESCRIPTION
**Pull request by SKaplanOfficial**
## Description

This extension adds commands for [Bike](https://hogbaysoftware.netlify.app/bike/). The following commands are provided:
- New Bike
  - Creates a new Bike document.
- New Bike from Clipboard
  - Creates a new Bike document from the clipboard contents.
- Append to Bike from Clipboard
  - Appends the clipboard contents to the end of the most recently active Bike document.
- New Bike from Template
  - Creates a new Bike document from a user-selected template.
- Minimize Bike
  - Minimizes all Bike windows.
- Close Other Bikes
  - Closes all Bike documents except the current one, asking whether to save them.
- Force Close Other Bikes
  - Closes all Bike documents except the current one, without saving them.
- Bike Background Color
  - Sets the background color of Bike windows.
- Bike Foreground Color
  - Sets the foreground color (text color) of Bike windows.
- Current Bike URL
  - Copies the URL of the current Bike document to the clipboard.

## Screencast

[Bike.webm](https://user-images.githubusercontent.com/7865925/188628421-5a3feca1-997c-4fe8-b091-3730f12e7e66.webm)

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by thomaspaulmann** [comment]
Oh cool, I've learned about Bike on a podcast. It's create to have an extension for it.

File: extensions/bike/package.json [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    },
+    {
+      "name": "set-foreground-color",
+      "title": "Bike Foreground Color",
+      "description": "Sets the foreground color of Bike windows.",
+      "mode": "view"
+    },
+    {
+      "name": "copy-url",
+      "title": "Current Bike URL",
```
Comment (pernielsentikaer) [inline extensions/bike/package.json:70]:
```suggestion
      "title": "Copy Bike URL",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for taking your time to look into this, I tested one of the commands and got this error:
CleanShot 2022-09-07 at 13 02 49@2x

It seems like it's not doing as expected? Happened to many commands and other didn't even do anything

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**